airflow/providers/amazon/aws/operators/bedrock.py

```diff
@@ -25,7 +25,10 @@
 from airflow.exceptions import AirflowException
```

Comment:
This comment is just a safety gate to make sure this one doesn't get merged out of order; anyone can feel free to resolve it once https://github.com/apache/airflow/pull/38849 is merged.

Reply:
I had to refactor the existing portion of the Bedrock system test, so this PR will need to be rebased on top of that and adjusted accordingly.  Do not merge until https://github.com/apache/airflow/pull/38887 is merged and rebased in.